Code/GraphMol/Depictor/Wrap/rdDepictor.cpp

```diff
@@ -101,18 +101,35 @@ unsigned int Compute2DCoordsMimicDistmat(
   return res;
 }
 
-void GenerateDepictionMatching2DStructure(RDKit::ROMol &mol,
-                                          RDKit::ROMol &reference, int confId,
-                                          python::object refPatt,
-                                          bool acceptFailure,
-                                          bool forceRDKit = false) {
+PyObject *GenerateDepictionMatching2DStructure(
+    RDKit::ROMol &mol, RDKit::ROMol &reference, int confId,
+    python::object refPatt, bool acceptFailure, bool forceRDKit,
+    bool allowRGroups) {
   RDKit::ROMol *referencePattern = nullptr;
   if (refPatt != python::object()) {
     referencePattern = python::extract<RDKit::ROMol *>(refPatt);
   }
+  auto matchVect = RDDepict::generateDepictionMatching2DStructure(
+      mol, reference, confId, referencePattern, acceptFailure, forceRDKit,
+      allowRGroups);
+  PyObject *atomMap = PyTuple_New(matchVect.size());
```

Comment:
What you have here isn't wrong, so this is more of a suggestion, but this code would be simpler if you used `python::tuple` instead of the raw Python C API

Reply:
Done.